apps/services/endorsements/api/src/app/modules/endorsementList/dto/validationRule.dto.ts

```diff
@@ -1,12 +1,15 @@
+import { ApiProperty } from '@nestjs/swagger'
 import { IsEnum, IsObject, IsOptional } from 'class-validator'
 import { IsEndorsementValidator } from '../../endorsementValidator/endorsementValidator.decorator'
 import { ValidationRule } from '../../endorsementValidator/endorsementValidator.service'
 export class ValidationRuleDto {
+  @ApiProperty({ enum: ValidationRule })
   @IsEnum(ValidationRule)
   type!: ValidationRule
 
+  @ApiProperty({ nullable: true })
```

Comment:
There's also `@ApiPropertyOptional()` but not sure if I prefer it to this.

Reply:
Makes me feel more consistent since nullable is used by the graphQL field decorators